scripts/stack/docker/docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+version: "3"
+
+networks:
+  connectors_stack_net:
+
+services:
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:${ELASTICSEARCH_VERSION}
+    environment:
+      - "discovery.type=single-node"
+      - "ES_JAVA_OPTS=-Xms2048m -Xmx2048m"
+      - "xpack.security.enabled=true"
+      - "xpack.security.authc.api_key.enabled=true"
+      - "xpack.security.authc.token.enabled=true"
+      - "ELASTIC_PASSWORD=${ELASTIC_PASSWORD}"
+      - "action.destructive_requires_name=false"
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+    networks:
+      - connectors_stack_net
+    ports:
+      - 9200:9200
+    volumes:
+      - es-data:/usr/share/elasticsearch/data
+
+  kibana:
+    image: docker.elastic.co/kibana/kibana:${KIBANA_VERSION}
+    ports:
+      - 5601:5601
+    depends_on:
+      - elasticsearch
+    networks:
+      - connectors_stack_net
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+    environment:
+      ELASTICSEARCH_URL: http://elasticsearch:9200
+      ELASTICSEARCH_HOSTS: http://elasticsearch:9200
+      ELASTICSEARCH_USERNAME: kibana_system
+      ELASTICSEARCH_PASSWORD: ${ELASTIC_PASSWORD}
+      XPACK_ENCRYPTEDSAVEDOBJECTS_ENCRYPTIONKEY: c69548d9027afcf4d55146b1d425a9f4c69548d9027afcf4d55146b1d425a9f4
+
+  elastic-connectors:
+    image: docker.elastic.co/enterprise-search/elastic-connectors:${CONNECTORS_VERSION}
+    depends_on:
+      - elasticsearch
+      - kibana
+    volumes:
+      - ${CURDIR}/connectors-config:/config
+    command: /app/bin/elastic-ingest -c /config/config.yml
+    network_mode: "host"
```

Comment:
Could you clarify why we need this? I think we no longer need it, but I may be wrong.

Reply:
This is needed as internally, the connectors config can use the other service names to communicate (e.g. "elasticsearch"), however, since we use the same configuration for configuring via the script as what is used in the container itself, we need this to be "localhost:9200" for communicating with the Elasticsearch node - having this makes it possible to use "localhost:9200" from both within the container as well as from our scripts.